src/static/js/sandboxLogrocket.js

```diff
@@ -0,0 +1,46 @@
+/* eslint no-undef: 0, no-console: 0 */
+
+function identifySession(currentUser) {
+  if (!currentUser || !Object.keys(currentUser).includes('email')) return;
+
+  if (window.LogRocket) {
+    window.LogRocket.identify(currentUser.email, {
+      name: currentUser.nickname || currentUser.name,
+      email: currentUser.email,
+
+      sandbox: true,
+    });
+  }
+}
+
+function getCurrentUserAndIdentifySession() {
+  const currentUserUrl = 'https://app.rookout.com/apiv1/current_user';
+
+  $.get({
+    url: currentUserUrl,
+    method: 'GET',
+    xhrFields: {
+      withCredentials: true,
+    },
+  }, (data) => {
+    identifySession(data);
+  })
+    .fail(() => {
+      console.warn("Currently not logged in to app.rookout.com");
+    });
+}
+
+function initLogrocket() {
+  if (window.LogRocket) {
+    window.LogRocket.init('fzkqiz/rookout');
+  }
+}
+
+
+$().ready(() => {
```

Comment:
I'd use `$(document).ready(() => ... )` instead, since the current form is deprecated (yes, I know it's just jQuery, but still 😂)